src/arkode/arkode_io.c

```diff
@@ -34,6 +35,360 @@
   ARKODE optional input functions
   ===============================================================*/
 
+/*---------------------------------------------------------------
```

Comment:
[Optional suggestion]: Move the new content in `arkode_io.c` to a separate file like `arkode_cli.c` since this file is already close to 4k lines